tests/analysis_tests/tests/src/expected/CompletionInferValues.res.txt

```diff
@@ -262,89 +310,112 @@ Path btn
   }, {
-    "label": "Js.String2.splitByReAtMost",
+    "label": "Int.toFixed",
     "kind": 12,
     "tags": [],
-    "detail": "(t, Js_re.t, ~limit: int) => array<option<t>>",
-    "documentation": {"kind": "markdown", "value": "\n`splitByReAtMost(str, regex, ~limit:n)` splits the given `str` at every\noccurrence of `regex` and returns an array of the first `n` resulting\nsubstrings. If `n` is negative or greater than the number of substrings, the\narray will contain all the substrings.\n\nSee [`String.split`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/split)\non MDN.\n\n## Examples\n\n```rescript\nJs.String2.splitByReAtMost(\"one: two: three: four\", /\\s*:\\s*TODO/, ~limit=3) == [\n    Some(\"one\"),\n    Some(\"two\"),\n    Some(\"three\"),\n  ]\n\nJs.String2.splitByReAtMost(\"one: two: three: four\", /\\s*:\\s*TODO/, ~limit=0) == []\n\nJs.String2.splitByReAtMost(\"one: two: three: four\", /\\s*:\\s*TODO/, ~limit=8) == [\n    Some(\"one\"),\n    Some(\"two\"),\n    Some(\"three\"),\n    Some(\"four\"),\n  ]\n```\n"}
+    "detail": "(int, ~digits: int=?) => string",
+    "documentation": {"kind": "markdown", "value": "\n`toFixed(n, ~digits=?)` return a `string` representing the given\nvalue using fixed-point notation. `digits` specifies how many digits should\nappear after the decimal point. See [`Number.toFixed`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed)\non MDN.\n\n## Examples\n\n```rescript\nInt.toFixed(123456) // \"123456.00\"\nInt.toFixed(10) // \"10.00\"\nInt.toFixed(300, ~digits=4) // \"300.0000\"\nInt.toFixed(300, ~digits=1) // \"300.0\"\n```\n\n## Exceptions\n\n- `RangeError`: If `digits` is less than 0 or larger than 100.\n"}
   }]
 
-Complete src/CompletionInferValues.res 53:130
-posCursor:[53:130] posNoWhite:[53:129] Found expr:[53:12->53:136]
-JSX <div:[53:12->53:15] onMouseEnter[53:16->53:28]=...[53:36->53:132]> _children:53:134
-posCursor:[53:130] posNoWhite:[53:129] Found expr:[53:36->53:132]
-posCursor:[53:130] posNoWhite:[53:129] Found expr:[53:41->53:130]
-posCursor:[53:130] posNoWhite:[53:129] Found expr:[53:123->53:130]
+Complete src/CompletionInferValues.res 50:103
+posCursor:[50:103] posNoWhite:[50:102] Found expr:[50:12->50:109]
+JSX <div:[50:12->50:15] onMouseEnter[50:16->50:28]=...[50:36->50:105]> _children:50:107
+posCursor:[50:103] posNoWhite:[50:102] Found expr:[50:36->50:105]
+posCursor:[50:103] posNoWhite:[50:102] Found expr:[50:41->50:103]
+posCursor:[50:103] posNoWhite:[50:102] Found expr:[50:95->50:103]
+Completable: Cpath Value[btn]->spl <<jsx>>
+Package opens Pervasives.JsxModules.place holder
+ContextPath Value[btn]->spl <<jsx>>
+ContextPath Value[btn]
+Path btn
+ContextPath Value[Int, toString](Nolabel)
+ContextPath Value[Int, toString]
+Path Int.toString
+[]
```

Comment:
This highlights that we've reduced the scope of the unsaved-completion-inference logic previously for pipes. In practice this does not matter though, because incremental type checking in the editor tooling takes care of this in a much better way.

Will remove these tests in another PR later on, but don't want to touch them for now.